test/libsolidity/semanticTests/array/copying/copy_dynamically_sized_storage_array.sol

```diff
@@ -0,0 +1,21 @@
+contract C {
+    function testFunction1() public {}
+    function testFunction2() public {}
+    function testFunction3() public {}
+
+
+    function() external [] externalArray0 = new function() external[] (3);
+    function() external [] externalArray1 = [this.testFunction1, this.testFunction2, this
+    .testFunction3];
```

Comment:
This is the way we usually wrap parameters, lists, etc. and I think it's more readable than wrapping it at a random spot in the expression:
```suggestion
    function() external [] externalArray1 = [
        this.testFunction1,
        this.testFunction2,
        this.testFunction3
    ];
```

Reply:
By the way, I think it' won't pass tests. We don't support dynamic array literals yet (#11879). To make it work you'll probably need to use a statically-sized array:
```suggestion
    function() external[3] externalArray1 = [this.testFunction1, this.testFunction2, this
    .testFunction3];
```